package.json

```
@@ -17,7 +17,7 @@
     "storybook:export": "yarn workspace storybook export",
     "deduplicate": "node scripts/deduplicate.js",
     "stylelint": "stylelint '**/*.js' '**/*.ts' '**/*.tsx'",
-    "prettier": "node ./scripts/prettier.js --branch master",
+    "prettier": "node ./scripts/prettier.js",
```

I would keep it explicit. If the core changes from next to master, it will break us.

```suggestion
    "prettier": "node ./scripts/prettier.js --branch next",
```